azure/azure.go

```diff
@@ -52,12 +57,49 @@ func (a *AzureClient) uploadFile(sessionLogger lager.Logger, containerReference
 		return fmt.Errorf("error in uploadFile could not open file: %w", err)
 	}
 	defer file.Close()
-
-	blob := containerReference.GetBlobReference(remoteFilePath)
-
-	err = blob.CreateBlockBlobFromReader(file, &storage.PutBlobOptions{})
+	stat, err := file.Stat()
 	if err != nil {
-		return fmt.Errorf("error in uploadFile could not create block: %w", err)
+		return fmt.Errorf("error in uploadFile could not get stats of file: %w", err)
+	}
+	blob := containerReference.GetBlobReference(remoteFilePath)
+	// single file limit
+	if stat.Size() < Limit {
```

Comment:
@JulianBenitez99 can we skip this if else block? Can you confirm if else block will work for file size < Limit?

Reply:
Confirmed on the pipeline that it works for size < Limit